pkg/config/config.go

```diff
@@ -14,12 +14,18 @@ import (
 )
 
 const (
-	configEnvName  = "ODOCONFIG"
-	configFileName = "odo"
+	globalConfigEnvName = "GLOBALODOCONFIG"
+	localConfigEnvName  = "LOCALODOCONFIG"
+	configFileName      = "odo-config"
```

Comment:
it might be better to have file with extension ¨odo-config.yaml` to show what format we are using

Reply:
ack